src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -115,6 +115,15 @@ function IOURequestStepConfirmation({
         IOU.setMoneyRequestBillable_temporaryForRefactor(transactionID, defaultBillable);
     }, [transactionID, defaultBillable]);
 
+    const defaultCategory = lodashGet(
+        _.find(lodashGet(policy, 'customUnits', {}), (customUnit) => customUnit.name === CONST.CUSTOM_UNITS.NAME_DISTANCE),
+        'defaultCategory',
+        '',
+    );
+    useEffect(() => {
+        IOU.setMoneyRequestCategory_temporaryForRefactor(transactionID, defaultCategory);
```

Comment:
This always resets category to default one while in money request flow.
This should be happen only when unset


https://github.com/Expensify/App/assets/108292595/67e6e842-9bd6-4165-a69b-70c710212149